data/projects.js

```diff
@@ -561,5 +561,26 @@ coala should support generating metrics for your code.",
 	  "tags" : ["Information Extraction", "CLI", "CI"],
 	  "markdown" : "enhance_coala_quickstart.md",
 	  "collaborating_projects" : ["coala"]
+	},
+	{
+		"name" : "Newcomer metrics and gamification",
```

Comment:
Gamification could be a problem. Many people pinging for reviews. People complaining that they did not get reviews on time and hence were delayed. People fighting over badges depending on the weight they carry. We had this in the uni system for a while with assignment submissions and the gamification turned into an absolute nightmare. We could have newcomers fighting to get things merged rather than actually focusing on the process of learning. Statistics are okay as a motivator but gamification should be given a rethink.